json/src/main/java/io/airlift/json/JsonSubType.java

```diff
@@ -53,6 +55,20 @@ public class JsonSubType
 {
     private final Set<Module> modules;
 
+    public static boolean isJsonSubType(ObjectMapper objectMapper, Object instance)
+    {
+        if (instance != null) {
```

Comment:
I'd invert this condition to reduce nesting:

if (instance == null) {
    return false;
}